src/frame/js/web-messenger.jsx

```diff
@@ -179,7 +179,7 @@ export function init(props = {}) {
             const {config: {app: {status}}} = store.getState();
 
             if (status !== 'active') {
-                return Promise.resolve();
+                throw new Error('App is inactive.');
```

Comment:
The call to `cleanUp` on line 218 fixes the case where you can still send a message to an inactive app, right?